src/scene-entry-manager.js

```diff
@@ -143,6 +143,7 @@ export default class SceneEntryManager {
   };
 
   _setupPlayerRig = () => {
+    this.playerRig.querySelector(".model").object3D.visible = true;
```

Comment:
You wont need this if you follow the above suggestion about preloading the avatar